src/routes/+layout.svelte

```diff
@@ -187,12 +193,13 @@
 	<main class="lg:ml-72 w-full mx-auto">
 		{#if $loginState === 'in'}
 			<slot />
-		{:else if $loginState === 'pending'}
-			<div class="m-auto flex items-center justify-center pt-10">
-				<Spinner />
-			</div>
 		{:else}
-			<div class="m-auto flex items-center justify-center pt-10">
+			{#if $loginState === 'pending'}
```

Comment:
This is a bit of hacky way to ensure that the FB SDK is loaded without actually displaying the login button if the user is already logged in (i.e. already has a valid cookie).

Reply:
Also on a side note, it would be nice if we could find a way to only load the FB SDK if the user actually needs to log in or log out (something most probably do very rarely).  This might provide a reduction in app initialization time?

Reply:
the benefit of that is very minimal. ideally this app would implement appropriate lazy-loading strategy where data is only loaded per component, but at the moment we kinda load it everywhere.